css/style.css

```diff
@@ -43,7 +38,6 @@ a:hover {
 }
 
 #nav li {
-    font-size: 1.3em;
```

Comment:
Why the font size reduction for nav elements?

Reply:
I don't feel like they need it. Their color and position are enough to indicate their function and having them the same font-size as the body creates a more uniform experience.